service/docs/source/geopm_pio.7.rst

```diff
@@ -81,6 +81,13 @@ Descriptions Of High Level Aliases
     <geopm_pio_cnl.7:Requirements>` and :doc:`geopm_pio_msr(7)
     <geopm_pio_msr.7>` for signal availability requirements.
 
+``BOARD_POWER_LIMIT_CONTROL``
+    The average board power usage limit over the time window specified in
+    platform PL1_TIME_WINDOW.
```

Comment:
Would it be better to reference the high-level alias instead (BOARD_POWER_TIME_WINDOW_CONTROL)?